src/main/java/picard/util/IntervalListTools.java

```diff
@@ -253,38 +253,68 @@
             IntervalList act(final List<IntervalList> list, final List<IntervalList> __) {
                 return IntervalList.concatenate(list);
             }
+
+            @Override
+            public boolean takesSecondInput() {
+                return false;
+            }
         },
         UNION("Like CONCATENATE but with UNIQUE and SORT implied, the result being the set-wise union of all INPUTS, " +
                 "with overlapping and abutting intervals merged into one.") {
             @Override
             IntervalList act(final List<IntervalList> list, final List<IntervalList> __) {
```

Comment:
I don't think `__` is normally used in java names (although I don't think it's completely disallowed as it is in C/C++). `unused` would be OK if we don't want to use `list2` here.